go/consensus/cometbft/apps/roothash/state/state_test.go

```diff
@@ -147,4 +202,75 @@ func TestSeparateRuntimeRoots(t *testing.T) {
 	ioRoot, err := st.IORoot(ctx, runtime.ID)
 	require.NoError(err, "IORoot")
 	require.EqualValues(blk.Header.IORoot, ioRoot)
+
+	roots, err := st.PastRoundRoots(ctx, runtime.ID)
+	require.NoError(err, "PastRoundRoots")
+	require.EqualValues(1, len(roots))
+	require.EqualValues(len(roots), st.PastRoundRootsCount(ctx, runtime.ID))
```

Comment:
```suggestion
	require.EqualValues(1, st.PastRoundRootsCount(ctx, runtime.ID))
```

Reply:
Your change is semantically different from what I want the test to actually test :)

Reply:
I just thought it is easier to read this way.

Reply:
The `PastRoundRootsCount` method and `len(roots)` are basically the same, except that the method is more optimized -- putting a hardcoded number in this check is slightly confusing to me.